backend/src/nodes/nodes/pytorch/convert_to_onnx.py

```diff
@@ -79,4 +79,4 @@ def run(self, model: PyTorchSRModel, is_fp16: int) -> Tuple[OnnxModel, str]:
 
         fp_mode = "fp16" if should_use_fp16 else "fp32"
 
-        return OnnxModel(onnx_model_bytes), fp_mode
+        return load_onnx_model(onnx_model_bytes), fp_mode
```

Comment:
Since the PyTorch model is guaranteed to be an SR model, shouldn't the ONNX model be one as well? If so, then the ONNX output of this node should reflect that.

(I added the comment here, because GH wouldn't allow me to near `self.outputs`.)

Reply:
What does PyTorch have to do with ONNX?

Reply:
From what I understand, a pytorch SR model could never be a ONNX rembg model.

Reply:
To expand on that, we've always allowed ONNX and NCNN to load any model so people aren't restricted by what we add in. So we don't guarantee SR models for anything. The only difference here is that for the Remove Background node, we need to be able to guarantee a valid model.

Reply:
> So we don't guarantee SR models for anything.

We guarantee SR models for PyTorch.

Reply:
I'm talking about ONNX, not PyTorch.

Reply:
But *I am* talking about PyTorch. This is the PyTorch to ONNX node.

Reply:
Ohhhh. I didn't see that. I thought you were commenting on the Load Model node.

Reply:
Hmm. I kind of don't want to make a specific SR model class there? Since the supported PyTorch SR models are a subset of possible ONNX SR models that a user could load. It would be a little strange to categorize some as Generic and some as SR. From the user end, it doesn't really matter whether the convert node guarantees something as "not rembg," but if I were to do that, I'd rather just specify it as Generic.

Reply:
> If I were to do restrict the output type, I think I'd rather just specify it as Generic.
That's what I wanted to suggest.